src/Header/Header.js

```diff
@@ -31,5 +32,9 @@ class Header extends Component {
   }
 }
 
+Header.propTypes = {
+  loggedIn: PropTypes.bool
+}
+
```

Comment:
I see that Header also gets 2 functions, updateLoginStatus & updateUserId, so I think we can add PropTypes.func for those, right?

Reply:
Good catch! I missed those two functions. I've now added them in.